pkg/model/uimodel.go

```diff
@@ -69,7 +78,7 @@ func (u *UIModel) Init() tea.Cmd {
 	return nil
 }
 
-var green = lipgloss.NewStyle().Foreground(lipgloss.Color("#04B575")).Render
```

Comment:
Why was this changed?

Reply:
I made this change to reflect the usage as well, i.e, red for highly utilized. I can add this too, as part of the flag if that works?

Reply:
Yes, I want to keep the default colors as-is and optionally allow someone to swap them.